rclip/main.py

```diff
@@ -216,7 +219,7 @@ def main():
     args.exclude_dir,
     args.no_indexing,
   )
-
+  XNVIEW_PATH = "C:\\Program Files\\XnViewMP\\xnviewmp.exe"
```

Comment:
1. What if I have xnview installed to a different path?
2. What if I don't have xnview installed?

Reply:
3. Why xnview? What about other image viewers.